ideas/co-design-draft-proposal/index.html

```diff
@@ -0,0 +1,176 @@
+               </span>
+           </div>
+       </div>
+       <!-- END markup for Preference Editor -->
+
+    <div id="ideasHeader">
+
+        <div class="ideasHeaderContainer">
+            <h1><a href="/">Co-designing Inclusive Cities</a></h1>
+        </div>
+    </div>
+
+    <div id="ideasContent">
+
+        <div class="flc-toc-tocContainer toc"> </div>
+
+        <div class="ideasContentContainer">
+            <div class="ideasContentBlock">
+                <span class="ideasDate">May 10, 2018</span>
+                <h2>IDRC Inclusive Co-Design Draft Proposal</h2>
+
+                <h3>Inclusive Cities Co-Design Toolkit</h3>
+                <p>The goal of this project is to create, document, and share an evolving toolkit of perspectives, activities, and measures that support the community-led design of civic infrastructure and connected cities. The toolkit is intended to be used by multiple participants, including Sidewalk Labs, Waterfront Toronto, and the residents of Toronto. It will be co-created by these stakeholders in a manner that reflects the diversity of Toronto, and will be posted openly so that others can use and adapt it.</p>
+
+                <p>This toolkit will be used to assist in the creation of Sidewalk Toronto's <em>Inclusion and Accessibility Principles</em>, which will be included in the Master Innovation and Development Plan for the Quayside neighbourhood.</p>
+
+                <p>Sidewalk's Principles are being developed in parallel with other aspects of the Master Innovation and Development Plan, and some of those parallel aspects are not (currently) public or are still under development. Therefore, the full details of the plan won't be public while the co-design is being done. Without knowing the full scope of Sidewalk Toronto, it will be difficult to come up with a comprehensive set of principles and we might miss important aspects to consider. We propose the co-creation of the toolkit as a resource that can be reused over time as more details become known and the implementation of the project takes shape.</p>
+
+                <p>This toolkit would be used to discover, evaluate and provide reusable co-design activities that can be applied throughout the Sidewalk Toronto project; from the development of the MIDP to the building and subsequent habitation of the Quayside site. It aims to cut across all the work streams defined by Sidewalk Toronto to ensure the public is fully engaged throughout the process of designing Quayside. It will also include community-led measures to assess the effectiveness of the proposed principles and evolve them once the project has been built and is being lived in.</p>
+
+                <p>During the creation of the <em>Inclusion and Accessibility Principles</em>, we may find that some are in opposition to others. The toolkit will provide a framework in which to resolve those conflicts and provide clarity on why and how the principles have been created.</p>
```

Comment:
Lisa pointed out that this might be a strong claim to make. I wonder if we can soften this in Simon-style: "The toolkit can help provide a framework in which to navigate potential conflicts..."?

Reply:
We should check with Sepi on the exact wording.

Reply:
Let's go with this: "During the creation of the <em>Inclusion and Accessibility Principles</em>, we may find that some are in opposition to others. The toolkit can help provide a framework in which to navigate potential conflicts and clarify why and how the principles have been created."